holidays/countries/ivory_coast.py

```diff
@@ -0,0 +1,128 @@
+        * <https://www.timeanddate.com/holidays/ivory-coast/>
+        * <https://holidayapi.com/countries/ci/>
+    """
+
+    country = "CI"
+    # %s (estimated).
+    estimated_label = tr("%s (estimé)")
+    # %s (observed).
+    observed_label = tr("%s (observé)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (observé, estimé)")
+    # Ivory Coast gained independence in 1960.
+    start_year = 1960
+    default_language = "fr"
+    supported_languages = ("en_CI", "en_US", "fr")
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        # the observed dates for the Ivory Coast's islamic holidays have been verified against
+        # local references (COSIM) and align with the default Umm al-Qura calculations.
+        IslamicHolidays.__init__(self, show_estimated=islamic_show_estimated)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, IvoryCoastStaticHolidays)
+        kwargs.setdefault("observed_rule", SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Secular Holidays.
+
+        # New Year's Day.
+        self._add_new_years_day(tr("1er janvier"))
+
+        # Labor Day.
+        self._add_observed(self._add_labor_day(tr("Fête du travail")))
+
+        # Independence Day
+        self._add_observed(self._add_holiday_aug_7(tr("Fête Nationale")))
+
+        # Introduced in 1996. See details: https://en.wikipedia.org/wiki/C%C3%B4te_d%27Ivoire#History
+        if self._year >= 1996:
+            # National Peace Day.
+            self._add_holiday_nov_15(tr("Journée Nationale de la Paix"))
+
+        # Christian Holidays
+
+        # Easter Monday.
+        self._add_easter_monday(tr("Lundi de Pâques"))
+        # Ascension Day.
+        self._add_ascension_thursday(tr("Jour de l’Ascension"))
+        # Pentecost Monday (Whit Monday).
+        self._add_whit_monday(tr("Lundi de Pentecôte"))
+        # Assumption Day.
+        self._add_assumption_of_mary_day(tr("Fête de l’Assomption"))
+        # All Saints' Day.
+        self._add_all_saints_day(tr("Fête de la Toussaint"))
+        # Christmas Day.
+        self._add_christmas_day(tr("Fête de Noël"))
+
+        # Islamic Holidays
+
+        # Eid al-Fitr.
+        self._add_eid_al_fitr_day(tr("Fête de fin du Ramadan"))
+        # Eid al-Adha.
+        self._add_eid_al_adha_day(tr("Fête de la Tabaski"))
+        # Day after Prophet's Birthday.
+        self._add_mawlid_day(tr("Lendemain de l’Anniversaire de la Naissance du Prophète Mahomet"))
+        # Day after Night of Destiny.
+        self._add_laylat_al_qadr_day(tr("Lendemain de la Nuit du Destin"))
```

Comment:
An extra line is typically inserted between holiday entries to improve code readability.
```suggestion
        # Easter Monday.
        self._add_easter_monday(tr("Lundi de Pâques"))

        # Ascension Day.
        self._add_ascension_thursday(tr("Jour de l’Ascension"))

        # Pentecost Monday (Whit Monday).
        self._add_whit_monday(tr("Lundi de Pentecôte"))

        # Assumption Day.
        self._add_assumption_of_mary_day(tr("Fête de l’Assomption"))

        # All Saints' Day.
        self._add_all_saints_day(tr("Fête de la Toussaint"))

        # Christmas Day.
        self._add_christmas_day(tr("Fête de Noël"))

        # Islamic Holidays

        # Eid al-Fitr.
        self._add_eid_al_fitr_day(tr("Fête de fin du Ramadan"))

        # Eid al-Adha.
        self._add_eid_al_adha_day(tr("Fête de la Tabaski"))

        # Day after Prophet's Birthday.
        self._add_mawlid_day(tr("Lendemain de l’Anniversaire de la Naissance du Prophète Mahomet"))

        # Day after Night of Destiny.
        self._add_laylat_al_qadr_day(tr("Lendemain de la Nuit du Destin"))
```